setup.py

```diff
@@ -23,7 +23,7 @@
         "slack_sdk==3.13.0",
```

Comment:
while we are here should we unpin some other requirements?

Reply:
I think we can bump slack_sdk to the newest version, it shouldn't break anything since we're looking at minors. tox still passed with this change.